homeassistant/components/iometer/quality_scale.yaml

```diff
@@ -0,0 +1,60 @@
+rules:
+  # Bronze
+  action-setup: done # does not apply
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done # does not apply
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup: done
```

Comment:
should this be `exempt`, since the integration does not register for any event?

Reply:
Yes I marked it as such

Reply:
Done